deploy/build.sh

```diff
@@ -91,15 +91,21 @@ buildClient(){
 	printf "module.exports = {\n  sha: \`%s\`,\n  branch: \`%s\`,\n  message: \`%s\`\n}" $TRAVIS_COMMIT "${TRAVIS_BRANCH//\`/\\\`}" "${TRAVIS_COMMIT_MESSAGE//\`/\\\`}" > ../version.js
 
 	cat ../version.js
+	SC_THEME_LIST=['default', 'brb', 'n21', 'open', 'thr', 'int' ];
```

Comment:
```suggestion
	SC_THEME_LIST=('default' 'brb' 'n21' 'open' 'thr' 'int' )
```
Define a shell Array